src/components/ItaliaTheme/Blocks/Calendar/Body.jsx

```diff
@@ -172,21 +172,35 @@ const Body = ({ data, block, inEditMode, path, onChangeBlock, reactSlick }) => {
   const NextArrow = (props) => {
     const { onClick, className } = props;
     return (
-      <Button outline color={'unset'} className={className} onClick={onClick}>
+      <Button
+        outline
+        color={'unset'}
+        className={className}
+        onClick={onClick}
+        aria-label={intl.formatMessage(messages.calendar_next_arrow)}
+      >
         <FontAwesomeIcon
-          title={intl.formatMessage(messages.calendar_next_arrow)}
+          aria-hidden={true}
           icon={['fas', 'chevron-right']}
+          title={intl.formatMessage(messages.calendar_next_arrow)}
         />
       </Button>
     );
   };
   const PrevArrow = (props) => {
     const { onClick, className } = props;
     return (
-      <Button outline color={'unset'} className={className} onClick={onClick}>
+      <Button
+        outline
+        color={'unset'}
+        className={className}
+        onClick={onClick}
+        aria-label={intl.formatMessage(messages.calendar_prev_arrow)}
+      >
         <FontAwesomeIcon
-          title={intl.formatMessage(messages.calendar_prev_arrow)}
+          aria-hidden={true}
           icon={['fas', 'chevron-left']}
+          title={intl.formatMessage(messages.calendar_next_arrow)}
```

Comment:
1) Siamo sicuri di aver bisogno di questo title dentro al pulsante? Non basta l'aria-label su button? Ho fatto un test veloce e lo SR legge solo l'aria-label
2) Se il title è giusto, il corretto qui sarebbe "_prev_arrow"

Reply:
Hai ragione sul punto 2, bisogna che lo correggiamo @deodorhunter, mi è sfuggito.

Per il punto 1, ahimè dobbiamo metterlo. Non serve a niente per lo screen reader ma il tool che usiamo per il test (MAUVE++) vuole le aria-label in tutte le icone, anche se hanno aria-hidden. È inutile, ma non possiamo farci niente.